weather_dl/download_pipeline/clients.py

```diff
@@ -21,10 +21,15 @@
 import logging
 import os
 import typing as t
+import urllib3
+import warnings
 
 import cdsapi
 from ecmwfapi import ECMWFService
 
+warnings.simplefilter(
+    "ignore", category=urllib3.connectionpool.InsecureRequestWarning)
+
```

Comment:
I'm not sure I understood your meaning. Are you suggesting this be added in other files too? I wasn't actually sure the best place to add it, so I can put it wherever you suggest.

Reply:
There's a CDS Client in this file, and that seems to be where the warnings are coming from. My suggestion was more or less: what if we put this on line 87 (the constructor of `CdsClient`)?

This way, it will only suppress these warnings. I can see an argument how we'd like to prevent this error trace form occurring everywhere, in which case we'd leave this line as is.